<launch>
  <node name="usm_stim300_driver" pkg="usm_stim300_driver" type="usm_stim300_driver_node" />
  <param name = "device_name" value="/dev/ttyUSB0" type="string" />
  <param name = "stanard_deviation_of_gyro" value="0.0004" type="double" />
  <param name = "stanard_deviation_of_acc" value="0.004" type="double" />
  <param name = "sample_rate" value="125" type="double" />
</launch>
